### ISSUE TITLE
LineData - labels not corresponding when too many entries

### ISSUE THREAD

**Issue by kusan-thana** [PhilJay/MPAndroidChart#1812]
Labels are not aligned with the entries when I have too many entries.
Here is how I set the labels :
     
```
   for (int i = 0; i < data.length; i++) {
            entries.add(new Entry((int)(data[i][position]), (int)data[i][0])); //entry(y,x)
            labels.add(String.valueOf((int) data[i][0]));
        }
```
LineDataSet dataset = new LineDataSet(entries, item.getTitle().toString());        
LineData plot = new LineData(labels, dataset);

As you can see the labels are the same as the x values.
I got this problem around 30000 entries.

![sans titre](https://cloud.githubusercontent.com/assets/14549979/15539339/16b2ea92-2282-11e6-8e1f-59ffb5099bc3.png)

(In this case, y and x are the same).

Also a constructor for LineData without labels would be nice (the X axis labels would then be just like the Y axis)

**Comment by kusan-thana** [comment]
Looks like I've got a problem with the decimal separator. 1180 should be 1,18, etc.

**Comment by Sevenupboy** [comment]
i also met this problem, through some test, i guess it takes place when some entry's x repeat or the space of entry has big different

**Comment by kusan-thana** [comment]
Problem solved in v3.0.0-beta1
